site/layouts/index.html

```diff
@@ -86,6 +78,30 @@ <h2 class="f2 b lh-title mb2 white">Our Board of Advisors</h2>
 {{ partial "blog" . }}
 </section>
 
+<section id="video">
+<div class="mhn1-l center mw7">
+  
+  <div class="w-100">
+
+    <div style="padding:56.25% 0 0 0;position:relative;"><iframe src="https://player.vimeo.com/video/281862263?title=0&byline=0&portrait=0" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe></div><script src="https://player.vimeo.com/api/player.js"></script>
```

Comment:
This is is just my opinion, but general rule of thumb is that video  content tends to be very engaging and should ideally be one of the first things a user sees when they visit the page are least just below the fold. Any particular reason for moving it so far down the page?

Reply:
That was my feeling as well. We got feedback saying the video disrupted the information flow and that they might not be hooked enough to watch the video yet.